### PR TITLE
The name of the PSI4 data dir changed

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: easybuild/easyblocks/p/psi.py [hunk 1]
```diff
@@ -154,7 +155,8 @@ def install_step(self):
         try:
             for subdir in ['obj', self.psi_srcdir]:
                 # copy symlinks as symlinks to work around broken symlinks
-                shutil.copytree(os.path.join(self.builddir, subdir), os.path.join(self.installdir, subdir), symlinks=True)
+                shutil.copytree(os.path.join(self.builddir, subdir), os.path.join(self.installdir, subdir),
+                                symlinks=True)
```
Comment (boegel) [inline easybuild/easyblocks/p/psi.py]: why the wrap? looked fine?

File: easybuild/easyblocks/p/psi.py [hunk 2]
```diff
@@ -175,12 +177,16 @@ def sanity_check_step(self):
...
         super(EB_PSI, self).sanity_check_step(custom_paths=custom_paths)
 
     def make_module_extra(self):
         """Custom variables for PSI module."""
         txt = super(EB_PSI, self).make_module_extra()
-        txt += self.module_generator.set_environment('PSI4DATADIR', os.path.join(self.installdir, 'share', 'psi'))
+        psi4datadir = glob.glob(os.path.join(self.installdir, 'share', 'psi*'))
+        if len(psi4datadir) != 1:
+            raise EasyBuildError("Could not determine the PSI4 data dir, there are multiple possibilities: %s"
+                                 % psi4datadir)
```
Comment (boegel) [inline easybuild/easyblocks/p/psi.py]: don't wrap, and use `, psi4datadir`, no need for the `% "..."`

File: easybuild/easyblocks/p/psi.py [hunk 3]
```diff
@@ -175,12 +177,16 @@ def sanity_check_step(self):
...
+        txt += self.module_generator.set_environment('PSI4DATADIR', psi4datadir[0])
```
Comment (boegel) [inline easybuild/easyblocks/p/psi.py]: I would do this:

```python
if len(psi4datadir) == 1:
    txt += self.module_generator.set_environment('PSI4DATADIR', psi4datadir[0])
else:
    raise EasyBuildError("...", psi4datadir)
```

Also, there could be no hits, so your "multiple possibilities" could be wrong/weird :)

File: easybuild/easyblocks/p/psi.py (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyblocks/p/psi.py]: 120 chars

File: easybuild/easyblocks/p/psi.py (hunk 2 above)
Comment (wpoely86) [inline easybuild/easyblocks/p/psi.py]: 120 chars

File: easybuild/easyblocks/p/psi.py (hunk 3 above)
Comment (wpoely86) [inline easybuild/easyblocks/p/psi.py]: sanity check makes sure that there is at least one

Comment (boegel) [inline easybuild/easyblocks/p/psi.py]: OK, sure, we we like to keep if conditions 'positive'

and you're really doing an if-else kind of thing here, so use `if`/`else`

File: easybuild/easyblocks/p/psi.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/p/psi.py]: ok

File: easybuild/easyblocks/p/psi.py (hunk 2 above)
Comment (boegel) [inline easybuild/easyblocks/p/psi.py]: ok, but use `, psi4datadir)`

File: easybuild/easyblocks/p/psi.py (hunk 3 above)
Comment (wpoely86) [inline easybuild/easyblocks/p/psi.py]: I really don't see why this would be better...

Comment (boegel) [inline easybuild/easyblocks/p/psi.py]: It's mostly about readability.

Comment (wpoely86) [inline easybuild/easyblocks/p/psi.py]: yes, so? My is: if not 1, stop right here. I don't see how it can get any more readable?

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**